### DISCUSSION TITLE
Problems with modals

### DISCUSSION THREAD

**Discussion by Trystanlm** [twbs/bootstrap#40588] [Q&A]
Hello, i just try to put a modal on my web site but i have a problems i let you see ; 
![image](https://github.com/twbs/bootstrap/assets/145838082/bb8b6d05-f5d4-4588-b60f-ad75b6dd875f)

When i click on my button, the modal is well opened but a grey background come and i cant interact with anything on my site .

So if u can help me im happy

**Comment by julien-deramond** [comment]
Hello @Trystanlm 

Could you please try to share some code to reproduce the issue, or maybe try to create a reproducible live example with [CodePen](https://codepen.io/team/bootstrap/pen/qBamdLj), StackBlitz, or any other tools?

Without any code, you could check things like that:
- Is your markup correct by comparing it to the documentation?
- Is the modal declared at the right place in the DOM?
- Are the backdrop options used correctly?
- Do you have errors in the console?
